CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+## v0.0.3
+
+*September 1, 2020*
+
+This release focuses on the IBC message processor framework (formerly known as handler framework) and initial implementations in ICS02 and ICS03. It also introduces an initial specification for the relayer algorithm.
```

Comment:
Maybe we need a line break here, only I couldn't figure out where exactly is the best.